docs/installation.md

```diff
@@ -194,7 +194,7 @@ freqtrade -c config.json
 
 #### 7. (Optional) Post-installation Tasks
 
-On Linux, as an optional post-installation task, you can setup the bot to run as a `systemd` service. See [Advanced Post-installation Tasks](advanced-setup.md) for details.
+On Linux, as an optional post-installation task, you may wish to setup the bot to run as a `systemd` service or configure it to send the log messages to the `syslog`/`rsyslog` or `journald` daemons. See [Advanced Post-installation Tasks](advanced-setup.md) for details.
```

Comment:
you may want to directlink this to the logging section by using the tag `advanced-setup.md#logging-to-syslog-or-journald` as link (be careful should you rename that title ... see main review comment)